src/converters/traits.rs

```diff
@@ -0,0 +1,12 @@
+use anyhow::{Error, Result};
+use rerun;
+use std::io::Cursor;
+use std::sync::Arc;
+
+pub trait Converter: Send + Sync {
+    fn convert(
+        &self,
+        rec: &Arc<rerun::RecordingStream>,
+        message: &mut Cursor<Vec<u8>>,
+    ) -> Result<(), Error>;
```

Comment:
```suggestion
    fn convert(
        &self,
        config: &Conversion,
        rec: &Arc<rerun::RecordingStream>,
        message: &mut Cursor<Vec<u8>>,
    ) -> Result<(), Error>;
```
I believe we should pass through the conversion config here, which is how the converter would know which entity-path to route to.